curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/match-characters-that-occur-zero-or-more-times.english.md

```diff
@@ -58,5 +58,9 @@ let result = chewieQuote.match(chewieRegex);
 
 ```js
 // solution required
-```
```

Comment:
actually, lines 62 to 65 should come in between line 60 and 61. Like this:

\``` js
let chewieQuote = "Aaaaaaaaaaaaaaaarrrgh!";
let chewieRegex = /A[a]*/; // Change this line
let result = chewieQuote.match(chewieRegex);
\```
We can also omit the comment: //solution required

Reply:
Hi @rupali317 thanks for the correction. Cheers.